code/datums/outfits/jobs/engineering.dm

```diff
@@ -1,6 +1,6 @@
 /decl/hierarchy/outfit/job/engineering
 	hierarchy_type = /decl/hierarchy/outfit/job/engineering
-	belt = /obj/item/weapon/storage/belt/utility/full
+	belt = /obj/item/weapon/storage/belt/utility/technomancer
```

Comment:
What's the point of renaming? It's still a full toolbelt, just with different contents. This will only cause conflicts for no reason.

Reply:
Resolved.